Documentation/Index.rst

```diff
@@ -0,0 +1,30 @@
+.. include:: Includes.txt
```

Comment:
I suggest to use the "official" documentation template:
https://github.com/TYPO3-Documentation/TYPO3CMS-Example-ExtensionManual/blob/master/Documentation/Index.rst